config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -508,3 +508,25 @@ periodics:
       - ../kubernetes/vendor/k8s.io/apimachinery/...
       - ../kubernetes/vendor/k8s.io/apiserver/...
       - ../kubernetes/vendor/k8s.io/client-go/...
+
+- name: ci-push-kubernetes-new-golang
```

Comment:
NIT: ci-build-and-push-k8s-at-golang-tip?

Reply:
Done - added appropriate testgrid dashboard annotation as well.